### ISSUE TITLE
Render the Union type with "or"

### ISSUE THREAD

**Issue by MathieuSchopfer** [agronholm/sphinx-autodoc-typehints#49]
When using a `Union` of types, e.g. `def my_func(arg: Union[float, int])`, the rendering is
> **arg** (Union[float, int])

Would it make sense to render this in plain English? e.g.
> **arg** (float or int)

nb: thanks for this great extension

**Comment by agronholm** [comment]
No, although in your case just `float` would suffice since it covers `int` too according to PEP 484.

**Comment by shongololo** [comment]
I agree with @MathieuSchopfer - the current formatting does not convey clear intent to readers of the docs. Can this please be reconsidered.

**Comment by untitaker** [comment]
I want to put down my vote for accepting alternative formatting styles, more generally. Current sphinx themes do not show complex signatures particularly well:

![Screenshot 2019-07-08 at 11 43 13](https://user-images.githubusercontent.com/837573/60800547-b1569d80-a175-11e9-8004-364602995ff5.png)

If sphinx-autodoc-typehints wrapped each part of the signature in appropriate CSS classes it would be easier to write JS/CSS to reformat this.
